frontend/src/features/projects/dispose/hooks/useStepForm.tsx

```diff
@@ -50,10 +51,13 @@ const useStepForm = () => {
   const onSubmitReview = (values: any, actions: any) => {
     const apiValues = _.cloneDeep(values);
     let response: any = dispatch(
-      updateWorkflowStatus(apiValues, values.statusCode, 'ACCESS-DISPOSAL'),
+      updateWorkflowStatus(apiValues, values.statusId, 'ACCESS-DISPOSAL'),
     );
     response
-      .then((values: any) => {
+      .then((values: IProject) => {
+        return dispatch(updateProject({ ...apiValues, rowVersion: values.rowVersion }));
```

Comment:
I don't like this, but currently I don't think there is a way to change a milestone status and call the update API in request.

Reply:
I'm not sure I understand.  For minor status updates or simply updating use the standard `update` endpoint.  If you are going to change a milestone status, then call the `setStatus` endpoint.  Do you mean you want to do both in a single request?

Reply:
So when the user clicks the "Approve" button the frontend must
1) Trigger a milestone status transition to the new status
2) Also save any changes that have been made to the form.

This requires two chained calls to the backend.